include/wren/JSHelper.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 1996-2021 Cyberbotics Ltd.
```

Comment:
This is a C file, so it should be lowercase

Reply:
Thanks, fixed in 42914ba